chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,604 @@
+    "\n",
+    "#### Posterior definition using the Bayes’ rule\n",
+    "The posterior is proportional to the product of **likelihood** and **prior**\n",
+    "\n",
+    "$$\\pi(\\mathbf{x} | b=b^\\mathrm{obs}) \\propto L (\\mathbf{x} | b=b^\\mathrm{obs}) \\pi(\\mathbf{x})$$\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In CUQIpy, we can use the class `BayesianProblem` to bundle the prior, the data distribution, and the data, then use it to explore the posterior distribution (e.g. point estimate and sampling):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BP = BayesianProblem(b, x)\n",
+    "print(BP)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we pass the data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BP.set_data(b=b_obs)\n",
+    "print(BP)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note the difference in the target of the `BayesianProblem` object before and after passing the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <font color=#CD853F> 2. Solving the BIP </font> <a class=\"anchor\" id=\"r-solving-the-bip\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 2.1. Maximum a posteriori (MAP) estimate </font> <a class=\"anchor\" id=\"r-maximum-a-posteriori-map-estimate\"></a>\n",
+    " \n",
+    "Maximizer of the posterior\n",
+    "\n",
+    "$$\\mathbf{x}^* = \\underset{\\mathbf{x}}{\\operatorname{argmax\\;}} \\pi(\\mathbf{x} | b=b^\\mathrm{obs})$$\n",
+    "In the case with Gaussian noise and Gaussian prior, this is the classic Tikhonov solution\n",
+    "\n",
+    "$$\\mathbf{x}^* = \\underset{\\mathbf{x}}{\\operatorname{argmin\\;}} \\frac{1}{2 \\sigma^2} ||\\mathbf{A}\\mathbf{x}- b^\\mathrm{obs}||_2^2 + \\frac{1}{2\\delta^2}||\\mathbf{x} ||^2_2$$\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To compute (and print) the MAP estimate in CUQIpy, we write:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "map_estimate = BP.MAP()\n",
+    "print(map_estimate)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 2.2. Sampling from the posterior </font> <a class=\"anchor\" id=\"r-sampling-from-the-posterior\"></a> \n",
+    "\n",
+    "The MAP point is a very useful point estimate, but it does not provide information about the uncertainty associated with the estimate. To quantify this uncertainty, we need to draw samples from the posterior distribution"
```

Comment:
Is it good to put the equation of the posterior distribution here, so the link between a MAP and posterior is more obvious? Just a thought.